src/Index.js

```diff
@@ -0,0 +1,321 @@
+const fs = require("fs");
+const path = require("path");
+
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const Manager = require("./lib/Manager");
+
+//  Starter questions
+const teamRole = [
+  {
+    type: "list",
+    name: "role",
+    message: "Which employee role do you want to add?",
+    choices: ["Manager", "Engineer", "Intern"],
+  },
+];
+
+// Questions to be rendered depending on the role chosen
+const managerQuestions = [
+  {
+    type: "input",
+    name: "name",
+    message: "Enter employee name:",
+  },
+
+  {
+    type: "input",
+    name: "id",
+    message: "Enter employee ID:",
+  },
+
+  {
+    type: "input",
+    name: "email",
+    message: "Enter employee email:",
+  },
+
+  {
+    type: "input",
+    name: "officeNumber",
+    message: "Enter manager's office number:",
+  },
+];
+
+const engineerQuestions = [
+  {
+    type: "input",
+    name: "name",
+    message: "Enter employee name:",
+  },
+
+  {
+    type: "input",
+    name: "id",
+    message: "Enter employee ID:",
+  },
+
+  {
+    type: "input",
+    name: "email",
+    message: "Enter employee email:",
+  },
+
+  {
+    type: "input",
+    name: "github",
+    message: "Enter engineer's github:",
+  },
+];
+
+const internQuestions = [
+  {
+    type: "input",
+    name: "name",
+    message: "Enter employee name:",
+  },
+
+  {
+    type: "input",
+    name: "id",
+    message: "Enter employee ID:",
+  },
+
+  {
+    type: "input",
+    name: "email",
+    message: "Enter employee email:",
+  },
+
+  {
+    type: "input",
+    name: "school",
+    message: "Enter Intern's school:",
+  },
+];
+
+const continueQuestion = {
+  type: "confirm",
+  message: "Do you want to add another employee?",
+  name: "quit",
+};
+
+let employeeArray = [];
+
+// Function to start run the application
+const start = async () => {
+  let inProgress = true;
+
+  while (inProgress) {
+    const { role } = await inquirer.prompt(teamRole);
+
+    if (role === "Manager") {
+      const { name, id, email, officeNumber } = await inquirer.prompt(
+        managerQuestions
+      );
+      const employee = new Manager(name, id, email, officeNumber);
+      employeeArray.push(employee);
+    } else if (role === "Engineer") {
+      const { name, id, email, github } = await inquirer.prompt(
+        engineerQuestions
+      );
+      const employee = new Engineer(name, id, email, github);
+      employeeArray.push(employee);
+    } else if (role === "Intern") {
+      const { name, id, email, school } = await inquirer.prompt(
+        internQuestions
+      );
+      const employee = new Intern(name, id, email, school);
+      employeeArray.push(employee);
+    }
+
+    const { quit } = await inquirer.prompt(continueQuestion);
+
+    if (!quit) {
+      inProgress = false;
+    }
+  }
+  writeToFile("src/index.html", generateTeam(employeeArray));
+};
+
+const generateCards = (team) => {
+  let cards = [];
+  for (let i = 0; i < team.length; i++) {
```

Comment:
You could use an array method here instead of a for loop. Try using the .map() method - array methods are the best choice for looping through and manipulating arrays as there are many different methods for a variety of different tasks.  See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) for more information about the map method.